proxy_agent/src/redirector.rs

```diff
@@ -244,6 +244,28 @@ pub fn get_ebpf_file_path() -> PathBuf {
     bpf_file_path
 }
 
+pub fn update_imds_redirect_policy(redirect: bool) {
+    #[cfg(windows)]
+    {
+        windows::update_imds_redirect_policy(redirect);
+    }
+    #[cfg(not(windows))]
+    {
+        linux::update_imds_redirect_policy(redirect);
+    }
+}
```

Comment:
Another way to do this without writing a wrapper function would be to do something like this:

```suggestion
#[cfg(windows)]
pub use windows::update_imds_redirect_policy;
#[cfg(not(windows))]
pub use linux::update_imds_redirect_policy;
```

You should also be able to use the `pub use linux::{func1, func2, ...};` syntax since it looks like there's a lot of these functions here.